src/reactA11yAnchorsRule.ts

```diff
@@ -95,6 +117,29 @@ class ReactA11yAnchorsRuleWalker extends ErrorTolerantWalker {
         }
     }
 
+    private compareAnchorsText(anchor1: IAnchorInfo, anchor2: IAnchorInfo): boolean {
+        let text1: string = anchor1.text;
+        let text2: string = anchor2.text;
+        let altText1: string = anchor1.altText;
+        let altText2: string = anchor2.altText;
+
+        if (this.ignoreCase) {
+            text1 = text1.toLowerCase();
+            text2 = text2.toLowerCase();
+            altText1 = altText1.toLowerCase();
+            altText2 = altText2.toLowerCase();
+        }
+
+        if (this.ignoreWhitespace) {
+            text1 = text1.trim();
+            text2 = text2.trim();
+            altText1 = altText1.trim();
+            altText2 = altText2.trim();
```

Comment:
Is it intentional that this only trims instead of replacing whitespace? Should `"a b"` still be considered different from `"ab"`?

I would have thought the rule should remove all whitespace? Perhaps not a blocker for this PR, but maybe the `ignore-whitespace` should be allowed to be `"all" | "trim"` to allow either behavior?

Reply:
The original issue is only concerned about leading/trailing whitespace, but I agree that there should be an option to enable either behavior. I will make some changes then.